confluent_platform/README.md

```diff
@@ -0,0 +1,100 @@
+# Agent Check: Confluent Platform
+
+## Overview
+
+This check monitors Confluent Platform and Kafka components through the Datadog Agent.
+
+This integration collects JMX metrics for the following components:
+- Broker
+- Connect
+- Replicator
+- Schema Registry
+- SQL Server
+- Streams
+- REST Proxy
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][1] for guidance on applying these instructions.
+
+### Installation
+
+The Confluent Platform check is included in the [Datadog Agent][2] package, so you don't need to install anything else on your Confluent Platform component server.
+
+The check collects metrics via JMX, so you need a JVM on each node so the Agent can run [jmxfetch][3]. We recommend using an Oracle-provided JVM.
+
+
+### Configuration
+
+1. Edit the `confluent_platform.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your confluent_platform performance data. See the [sample confluent_platform.d/conf.yaml][4] for all available configuration options.
+
+    For each component, you need to create a separate instance to collect its jmx metrics. List of default metrics collected are listed in [`metrics.yaml` file][9]. For example:
+
+    ```yaml
+    instances:
+     - host: localhost
+       port: 8686
+       name: broker_instance
+       user: username
+       password: password
+     - host: localhost
+       port: 8687
+       name: schema_registry_instance
+     - host: localhost
+       port: 8688
+       name: rest_proxy_instance
+    ```
+
+2. [Restart the Agent][5].
+
+#### Containerized
+
+##### Metric collection
+
+For containerized environments, see the [Autodiscovery with JMX][10] guide.
+
+### Validation
+
+[Run the Agent's status subcommand][6] and look for `confluent_platform` under the **JMXFetch** section.
+
+    ========
```

Comment:
This should be within a code block instead of just being indented

Reply:
Sure, done.